### PR TITLE
Momchil/updates

### PR DESCRIPTION
**Pull request by momchil-flex**
- Removed load_without_validation
- A simulation with all components is now uniquely defined in `tests/utils.py`. There was another simulation used for the updater and it was missing components, which made us miss an updater bug for `ModeFieldMonitor` -> `ModeSolverMonitor`.
- Adding the `Near2FarKSpaceMonitor` to the full simulation breaks the hdf5 IO, but this will be refactored soon so I commented it out for now.

### CODE REVIEW

File: tests/utils.py [hunk 1]
```diff
@@ -183,6 +183,36 @@ def prepend_tmp(path):
...
+            size=(2, 2, 2),
+            freqs=[250e12, 300e12],
+            name="n2f_cartesian",
+            custom_origin=(1, 2, 3),
+            x=[-1, 0, 1],
+            y=[-2, -1, 0, 1, 2],
+            plane_axis=2,
+            plane_distance=5,
+        ),
+        # Near2FarKSpaceMonitor(
```
Comment (tylerflex) [inline tests/utils.py:206]: why was this removed out of curiosity?

**Review by tylerflex — APPROVED** [review]

File: tests/utils.py (hunk 1 above)
Comment (momchil-flex) [reply tests/utils.py:206]: Oh I commented on that in the PR description, it breaks the hdf5 IO test... but this will be reorganized *again* soon so I left it for now.
